src/content/docs/tutorial-optimize-telemetry/detect-anomalies.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: "Detect ingest anomalies"
+metaDescription: Use your baseline report to detect ingest anomalies.
+redirects:
+---
+
+import omaoedgIngestTargetLine from 'images/oma-oe-dg_screenshot-full_ingest-target-line.webp'
+
+import omaoedgLookoutViewConsumingAccount from 'images/oma-oe-dg_screenshot-crop_lookout-view-consuming-account.webp'
+
+Once you've installed the [baseline dashboard](/docs/tutorial-optimize-telemetry/create-baseline-report/), you should keep an eye on it to detect any ingest anomalies as often as possible. While you have to wait until you have enough data to optimize your ingest, you can use your dashboard to find issues with your ingest much sooner. You can use the baseline dashbaord as is, or download optional dashboards to give you even greater anomaly detection options.
+
+You should also set up alerts to help notify you of issues as they occur. You can use our [ingest alerts guide](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts) to create a large variety of alerts, but we recommend you set up these two alerts as a minimum:
+
+* A threshold alert to notify if you exceed monthly targets for data ingest beyond seasonal increases
+* An anomaly alert to notify you of a sudden sharp increase ingest data
+
+<Callout variant="tip">
+New to creating alerts with our platform? Check out our [alert creation and management tutorial](/docs/tutorial-create-alerts/create-new-relic-alerts/) to learn everything you need about creating New Relic alerts.
+</Callout>
+
+## Use the Lookout view to monitor anomalies [#lookout]
+
+The baseline dashboard you set up in the previous step of this tutorial has a feature called **Lookout**. In a nutshell, this feature lets you provide a NRQL query to instantly search for ingest anomalies over any period of time. For example, the following NRQL query will give you information on all ingest anomalies by `usageMetric` over the last 24 hours:
```

Comment:
Maybe we can reframe this to be more user-centric, e.g., 'After installing the dashboard, you can get a high level view of your ingest patterns. Going to Lookout, you can run NRQL queries to search for ingest anomalies over time. For example,'

Reply:
(Riffing on the nouns here)

Reply:
Good callout, I've played with the words a bit and rephrased this. Thank you!